src/UI/examples/Button/Standard/with_js_binding.php

```diff
@@ -0,0 +1,14 @@
+<?php
+function with_js_binding() {
+    global $DIC;
+    $f = $DIC->ui()->factory();
+    $renderer = $DIC->ui()->renderer();
+
+    return $renderer->render(
+		$f->button()->standard("Goto ILIAS", "#")
```

Comment:
No, "" will reload the page, which probably destroys exactly what you tried to achieve with your action.